firefox-ios/Client/Frontend/Home/LogoHeader/HomeLogoHeaderCell.swift

```diff
@@ -112,6 +112,7 @@ class HomeLogoHeaderCell: UICollectionViewCell, ReusableCell {
 // MARK: - ThemeApplicable
 extension HomeLogoHeaderCell: ThemeApplicable {
     func applyTheme(theme: Theme) {
+        // TODO: This can be moved to the new homescreen wallpaper fetching redux
```

Comment:
Is there a ticket number for this task? Could add it here, but if you're the one working on it it's probably also fine not too.